inference-engine/src/mkldnn_plugin/memory_descs/mkldnn_memory_desc.h

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "cpu_blocked_memory_desc.h"
+
+/**
+ * @brief
+ *
+ * MKLDNNMemoryDesc - the descriptor of tensor representation in memory. Describes all required information
+ * for proper allocation and handling tensor in some buffer. The real memory is not present, just description.
+ * This object answers on question how and where data with logical index [x1, x2, .. xN] placed in real buffer.
+ * In the simplest case it describe a mapping between "logical offset" and "real offset".
+ *
+ */
+
+namespace MKLDNNPlugin {
+
+/**
+ * Represent internal plugin abstraction of tensor description
+ *
+ */
+class MKLDNNMemoryDesc : public virtual MemoryDesc {
```

Comment:
M.b. rename MKLDNNMemoryDesc to OnednnMemoryDesc since we introduced OnednnBlockedMemoryDesc